src/adapters/pois_styles.js

```diff
@@ -2,10 +2,9 @@
 export const filteredPoisStyle = {
   type: 'symbol',
   layout: {
-    'icon-image': 'pin_with_dot',
+    'icon-image': ['concat', 'pin-', ['get', 'subclass']],
```

Comment:
The icon associated to the POI is not always identical to the subclass name.

Either this layer should integrate the same kind of `icon-image` rules (that are quite long), currently generated by the "icons" task in the map-style-builder to build other POIs layers used by vector tiles.
Or probably simpler, `poisToGeoJSON` should use the IconManager to evaluate the icon associated with each poi, and store it as an extra GeoJSON property.